### PR TITLE
Update installation.md

### PR DESCRIPTION
**Pull request by aeisenberg**
Add some troubleshooting on the `make dev` command.

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -143,3 +154,43 @@ Login username is `admin` and the password is `admin`.
 ### Elasticsearch access via ElasticHQ
 
 [elastichq](https://hub.docker.com/r/elastichq/elasticsearch-hq/) Access at [localhost:5000/](http://localhost:5000/)
+
+## Fixing `make dev` errors
```
Comment (comzeradd) [inline docs/installation.md:158]: Can you move this section under `Troubleshooting` as subsection? (`###`).

File: docs/installation.md [hunk 2]
```diff
@@ -143,3 +154,43 @@ Login username is `admin` and the password is `admin`.
...
+This error means that there is a process that is already registered to use port `8080`. It is most likely a running docker container that is using this port, but to check, run this command:
+
+```bash
+lsof -nP -iTCP -sTCP:LISTEN | grep 8080
+```
+
+If result will be something like this:
+
+```bash
+com.docke  5086 andrew.eisenberg   84u  IPv6 0xdc100c215fbb6b93      0t0  TCP *:8080 (LISTEN)
```
Comment (comzeradd) [inline docs/installation.md:178]: Could you anonymize that?
eg.
```
com.docke  5086 <username>   84u  IPv6 0xdc100c215fbb6b93      0t0  TCP *:8080 (LISTEN)
```

**Review by comzeradd — CHANGES_REQUESTED** [review]
Thank you for this contribution! 💚 

Looks good overall. Just added some minor comments.

**Comment by aeisenberg** [comment]
Addressed your changes and also added the extra dependencies required for the commit hooks.

File: .python-version [hunk 3]
```diff
@@ -0,0 +1 @@
+3.5.7
```
Comment (comzeradd) [inline .python-version:1]: I assume this file was checked in by mistake? Since we don't require a specific python version.

Comment (aeisenberg) [reply .python-version:1]: Thanks, yes. Let me remove that.

**Review by comzeradd — APPROVED** [review]
💯

**Pull request merged**